src/CommandLineUtils/HelpText/DefaultHelpTextGenerator.cs

```diff
@@ -209,27 +216,27 @@ public virtual void Generate(CommandLineApplication application, TextWriter outp
         /// <param name="output">Help text output</param>
         /// <param name="visibleCommands">Commands not hidden from help text</param>
         /// <param name="firstColumnWidth">The width of the first column of commands, arguments, and options</param>
+        /// <param name="descriptionFormatter">A description formatter for wrapping the descriptions</param>
         protected virtual void GenerateCommands(
             CommandLineApplication application,
             TextWriter output,
             IReadOnlyList<CommandLineApplication> visibleCommands,
-            int firstColumnWidth)
+            int firstColumnWidth,
+            DescriptionFormatter descriptionFormatter)
```

Comment:
Adding new parameters to `protected` methods is technically a breaking change. Anyone who inherits from DefaultHelpTextGenerator and overrides this method will be broken by this. Can you leave the method calls as-is? You could share the instance of descriptionFormatter between methods as a field or property on the class. If you make it a protected property, then anyone inheriting from DefaultHelpTextGenerator could take advantage of this helper class.

```
protected virtual DescriptionFormatter DescriptionFormatter { get; set; }
```